app/assets/javascripts/components/user_profiles/instructor_stats.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// import ByStudents from './by_students'
+// import StudentStats from './student_stats.jsx';
+
+const InstructorStats = React.createClass({
+  render() {
+    return (
+      <div>
+        <h5> InstructorStats by </h5>
+        <div className= "user_stats">
+          <div className= "stat-display">
```

Comment:
And if the `InstructorStats` and `StudentStats` components look dumb to me. You could make them functional components as well.

@sejalkhatri @ragesoss What do you guys think?